pkg/common/idutil/safety_test.go

```diff
@@ -166,6 +174,10 @@ func testCommonCheckIDNormalization(assertGood func(string), assertBad func(stri
 	assertGood("spiffe://example.org/workload/%23")
 	assertGood("spiffe://example.org/workload/%00")
 	assertGood("spiffe://example.org/workload/" + url.PathEscape("%E4%B8%96%E7%95%8C"))
+	assertGood("spiffe://example.org/workload/%E4%B8%96%E7%95%8C")
+	assertGood("spiffe://世界/workload")
+	assertGood("spiffe://example.org/世界")
+	assertGood("spiffe://%E4%B8%96%E7%95%8C/workload")
```

Comment:
I don't think we have any assertions that scheme rules are reverted with the safety valve?

Reply:
:+1 I'll add some.